com.unity.sg2/Editor/GraphDeltaRegistry/Registry/ContextBuilder.cs

```diff
@@ -36,7 +36,7 @@ public ShaderFunction GetShaderFunction(NodeHandler node, ShaderContainer contai
             // Cannot get a shader function from a context node, that needs to be processed by the graph.
             // -- Though, see comment before this one, it could do more- but it'd be kinda pointless.
             // It's also pointless unless a similar strategy is taken for Reference nodes- who also need a fair amount of graph processing to function.
-            throw new NotImplementedException();
+            return ShaderFunction.Invalid;
```

Comment:
This seemed like the easiest way to appease the preview system, although I'm not sure if it has unwanted implications. I didn't build out a workaround because contexts will eventually get their own node UI without previews.

Reply:
Ah-- it's okay. The ContextBuilder is a bit of an artifact and might get some changes. What we realized is that the context node needs graph level awareness and also has no real functional behavior or transformations apart from the connections feeding into it-- so all that behavior is in the interpreter.cs.

This should be fine for now.